### PR TITLE
Added unit test

### PR DESCRIPTION
**Pull request by MiladMoulayi**
Created unit test for ProductData component of Overview module.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -7,7 +7,6 @@ module.exports = {
   extends: [
     'plugin:react/recommended',
     'airbnb',
-    'airbnb/hooks',
```
Comment (gocodezhang) [inline .eslintrc.js]: Adding back airbnb/hooks

File: server/index.js
Comment (gocodezhang) [inline server/index.js:1]: There seems to be quite a few differences in server/controllers and index.js files

File: client/src/components/related-items-and-outfit/index.jsx
Comment (gocodezhang) [inline client/src/components/related-items-and-outfit/index.jsx:1]: There seems to be a few changes in related items module

**Review by gocodezhang — CHANGES_REQUESTED** [review]
Given there are a few differences in server folder and related item module, **suggesting a group discussion before merging**

**Review by HeyMaximus — APPROVED** [review]

**Review by gocodezhang — APPROVED** [review]

**Pull request merged**